### PR TITLE
Refresh CheckUpdates widget update count after execute

### PR DESCRIPTION
**Pull request by elParaguayo**
Updated pull request for automatic refreshing of update count following execution of update.

### CODE REVIEW

File: libqtile/widget/check_updates.py [hunk 1]
```diff
@@ -38,7 +38,8 @@ class CheckUpdates(base.ThreadedPollText):
         ("colour_no_updates", "ffffff", "Colour when there's no updates."),
         ("colour_have_updates", "ffffff", "Colour when there are updates."),
         ("restart_indicator", "", "Indicator to represent reboot is required. (Ubuntu only)"),
-        ("no_update_string", "", "String to display if no updates available")
+        ("no_update_string", "", "String to display if no updates available"),
+        ("refresh_after_execute", True, "Refresh update count after execute")
```
Comment (tych0) [inline libqtile/widget/check_updates.py:42]: Can we drop this config option? It's not clear to me when you'd ever not want to do this.

**Comment by tych0** [comment]
looks like there's some linting errors.

**Comment by elParaguayo** [comment]
I've removed the unused import line but I'm also getting errors for multiple lines being too long. However, these are existing lines in the code so I've assumed they're ok.

**Comment by tcld** [comment]
I'll check out this branch and wait for the next updates to come down the line. The code looks ok to me.

**Pull request closed without merging**